components/tikv_util/src/buffer_vec.rs

```diff
@@ -22,7 +22,7 @@ impl std::fmt::Debug for BufferVec {
             if item.is_empty() {
                 write!(f, "null")?;
             } else {
-                write!(f, "{}", hex::encode_upper(item))?;
+                write!(f, "{}", crate::hex_encode_upper(item))?;
```

Comment:
this needs to redact.

Reply:
This will cause cycle dependency between `tikv_util` and `log_wrappers`.

Reply:
Maybe we can move `hex_encode_upper` to `log_wrappers` and reexport it in `tikv_util`.